src/Sulu/Bundle/CategoryBundle/Controller/CategoryController.php

```diff
@@ -207,21 +177,11 @@ public function postAction(Request $request)
      */
     public function putAction($id, Request $request)
     {
-        try {
-            if (!$request->get('name')) {
-                throw new MissingArgumentException(self::$entityName, 'name');
-            }
-
-            return $this->saveEntity($request, $id);
-        } catch (MissingArgumentException $exc) {
-            $view = $this->view($exc->toArray(), 400);
-
-            return $this->handleView($view);
-        }
+        return $this->saveCategory($request, $id);
     }
 
     /**
-     * Partly changes an existing category.
+     * Updates the category which is assigned to the given id.
```

Comment:
Yeah, we use it like that in other controllers. And it really doesn't replace it, since it only sets some values on the same object instance. Replace would mean for me that it creates a new instance.